app/journal/journal-client.tsx

```
@@ -0,0 +1,252 @@
+            const searchLower = searchQuery.toLowerCase();
+            const matchesSearch =
+                trade.symbol.toLowerCase().includes(searchLower) ||
+                trade.side.toLowerCase().includes(searchLower) ||
+                trade.strategy.some(s => s.toLowerCase().includes(searchLower));
+            const matchesStrategy = filterStrategy === 'all' || trade.strategy.includes(filterStrategy);
+            const matchesHidden = showHidden || !trade.isHidden;
+            return matchesSearch && matchesStrategy && matchesHidden;
+        });
+
+        filtered.sort((a, b) => {
+            let aVal: any, bVal: any;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)